### ISSUE TITLE
Add param count check 

### ISSUE THREAD

**Issue by rokath** [rokath/trice#150]
When during `trice u` a trice statement  ist evaluated,  the format specifier count  should  match the  param count and **_n**, if specified. If not, report an error and write *#error* in front of trice statement. This allows generally  not to use **_n**.

**Comment by th-sabik** [comment]
A runtime check exists now, but a pre-compiletime check (`trice -u`) is higher quality

**Comment by rokath** [comment]
Merged with #291

**Comment by rokath** [comment]
For example this line caused a problonly at runtime:
```C
trice( iD( 1278), "diag:RTT0_bytesInBufferMax=%u, RTT0_writeSpaceMin\n", RTT0_bytesInBufferMax, RTT0_writeSpaceMin );
```

**Comment by rokath** [comment]
Write tests for `trice i -addParamCount` and `trice u -addParamCount` for TRICE, TRICE8,... ,trice64.

**Comment by rokath** [comment]
parameter count check is implemented in v0.67.0
